src/form-host/form-host.html

```diff
@@ -2,20 +2,33 @@
 
 <head>
   <title>xForm Test Host</title>
-  <link rel="stylesheet" type="text/css" href="../../dist/inbox.css" />
```

Comment:
is this intentional to use ext instead of dist? seems fine - but just confirming. maybe risk of something not working after npm?

Reply:
cool. i'm happy with it then